file_app/management/commands/load_files.py

```diff
@@ -0,0 +1,65 @@
+from django.core.management.base import BaseCommand
+from file_app.models import FileData
+from django.core import exceptions
+import os
+import csv
+
+
+# ========================================== Command Class inheriting BaseCommand ==============================
+class Command(BaseCommand):
+    help = "Loading files to the database"
+
+    """ this function handling the all operations"""
+    def handle(self, *args, **options):
+
+        """ here you have to set you directory(folder) path of you machine """
+        path = 'C:\\Users\\Muhammad Noman\\Desktop\\weatherman-master\\weatherfiles'
+        files = []
+
+        """ this loop getting all the files from path(directory) and storing into files list """
+        for r, d, f in os.walk(path):
+            for file in f:
```

Comment:
use better and meaningful variable names, see [pep8](https://www.python.org/dev/peps/pep-0008/) to make sure your written code is better.